go.mod

```diff
@@ -4,12 +4,12 @@ go 1.13
 
 require (
 	github.com/ONSdigital/dp-api-clients-go v1.7.0
-	github.com/ONSdigital/dp-graph/v2 v2.2.0
-	github.com/ONSdigital/dp-healthcheck v1.0.3
+	github.com/ONSdigital/dp-graph/v2 v2.2.2
+	github.com/ONSdigital/dp-healthcheck v1.0.5
```

Comment:
I think we now have some dependencies that use the ContextKey from go-ns and others that use the ContextKey from dp-net. I don't know if this would have any effect on this service, but I think it would be a good idea to upgrade all dependencies that depend on dp-net:
- dp-api-clients-go
- dp-kafka

And also, we should replace the go-ns/server in in main.go for dp-net/http server.